bundles/org.openhab.voice.openaitts/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="voice:openaitts">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication for connecting to OpenAI API.</description>
+		</parameter-group>
+		<parameter-group name="tts">
+			<label>TTS Configuration</label>
+			<description>Configure Text to Speech.</description>
+		</parameter-group>
+		<parameter name="apiKey" type="text" required="true" groupName="authentication">
+			<label>API key</label>
+			<required>true</required>
+			<description>OpenAI API key.</description>
+		</parameter>
+		<parameter name="apiUrl" type="text" required="true" groupName="authentication">
+			<label>Api URL</label>
+			<required>true</required>
+			<description>TTS host API URL.</description>
+			<default>https://api.openai.com/v1/audio/speech</default>
+		</parameter>
+		<parameter name="model" type="text" required="true" groupName="tts">
+			<label>Model</label>
+			<required>true</required>
+			<description>ID of the model to use.</description>
+			<default>tts-1</default>
+		</parameter>
+		<parameter name="voice" type="text" groupName="tts">
+			<label>Voice</label>
+			<required>true</required>
+			<description>The voice to use when generating the audio. Previews of the voices are available in the text to speech
+				guide. https://platform.openai.com/docs/guides/text-to-speech/voice-options</description>
+			<default>nova</default>
+		</parameter>
```

Comment:
Would be nice if you can select this in the UI by providing the options:

```xml
<options>
	<option value="alloy">Alloy</option>
	<option value="echo">Echo</option>
	<option value="fable">Fable</option>
	<option value="nova">Nova</option>
	<option value="shimmer">Shimmer</option>
	<option value="onyx">Onyx</option>
</options>
```

You can also allow other values by adding:

```xml
<limitToOptions>false</limitToOptions>
```

That way it is still possible to use other options if they become available in the future without first having to update the add-on.